### ISSUE TITLE
Installation on Windows 11 fails

### ISSUE THREAD

**Issue by DierkDroth** [dotnet/ResXResourceManager#493]
I'm trying to install the standalone app on my Windows 11 machine but am getting error below. I'm the admin of that machine and am not aware that I would have set any restrictions related.

Running setup.exe as Administrator fails with similar error.

Any idea what's going wrong?

<img width="427" alt="image" src="https://user-images.githubusercontent.com/26179242/184875210-7dec05d7-0b09-41f1-9483-52e7538ead62.png">

**Comment by tom-englert** [comment]
Exactly what's written in the dialogue.
Just adjust the security settings.
Or download the sources and build your own binaries.

**Comment by DierkDroth** [comment]
@tom-englert thanks for your feedback.

Like I said: I did not change any of the security settings on my Windows 11. Hence, I would not even know what settings needed to be tweaked in order to get the installer running.

Also, I tried to build the sources which fails with this error.
Severity	Code	Description	Project	File	Line	Suppression State
Error	APPX3217	SDK folder containing 'UAP.props' for 'UAP 10.0.18362.0' cannot be located. See http://go.microsoft.com/fwlink/?LinkID=798187 for more information.	ResXManager.Package	C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Microsoft\VisualStudio\v17.0\AppxPackage\Microsoft.AppXPackage.Targets	1097
